todo-list/src/Component/Todo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useRef, useEffect } from 'react';
```

Comment:
Please remove all unused imports. and put a new line between import and component.